netbox_onboarding/tests/test_onboard.py

```diff
@@ -119,8 +119,9 @@ def test_ensure_device_instance_not_exist(self):
         nbk.device_type = self.device_type1
         nbk.netdev.ot = self.onboarding_task3
 
-        nbk.ensure_device_instance()
+        nbk.ensure_device_instance(default_status="planned")
         self.assertIsInstance(nbk.device, Device)
+        self.assertEqual(nbk.device.status, "planned")
         self.assertEqual(nbk.device, nbk.netdev.ot.created_device)
         self.assertEqual(nbk.device.serial, "123456")
```

Comment:
If you accept my previous comment, then it'd make sense to check `nbk.device.device_role`, `nbk.device.device_type`, `nbk.device.platform` here as well.